packages/tpetra/core/src/Tpetra_Import_def.hpp

```diff
@@ -128,138 +130,169 @@ namespace Tpetra {
-  Import (const Teuchos::RCP<const map_type >& source,
+  Import(const Teuchos::RCP<const map_type >& source,
           const Teuchos::RCP<const map_type >& target,
           const Teuchos::RCP<Teuchos::ParameterList>& plist) :
-    out_ (Teuchos::getFancyOStream (Teuchos::rcpFromRef (std::cerr))),
-    debug_ (tpetraImportDebugDefault)
+    out_(Teuchos::getFancyOStream (Teuchos::rcpFromRef (std::cerr))),
+    debug_(tpetraImportDebugDefault)
   {
     Teuchos::Array<int> dummy;
-    init (source, target, false, dummy, plist);
+    init(source, target, false, dummy, plist);
   }
 
   template <class LocalOrdinal, class GlobalOrdinal, class Node>
   Import<LocalOrdinal,GlobalOrdinal,Node>::
-  Import (const Teuchos::RCP<const map_type >& source,
+  Import(const Teuchos::RCP<const map_type >& source,
           const Teuchos::RCP<const map_type >& target,
           const Teuchos::RCP<Teuchos::FancyOStream>& out,
           const Teuchos::RCP<Teuchos::ParameterList>& plist) :
-    out_ (out),
-    debug_ (tpetraImportDebugDefault)
+    out_(out),
+    debug_(tpetraImportDebugDefault)
   {
     Teuchos::Array<int> dummy;
-    init (source, target, false, dummy, plist);
+    init(source, target, false, dummy, plist);
   }
 
   template <class LocalOrdinal, class GlobalOrdinal, class Node>
   Import<LocalOrdinal,GlobalOrdinal,Node>::
-  Import (const Teuchos::RCP<const map_type >& source,
+  Import(const Teuchos::RCP<const map_type >& source,
           const Teuchos::RCP<const map_type >& target,
-          Teuchos::Array<int> & remotePIDs) :
-    debug_ (tpetraImportDebugDefault)
+         Teuchos::Array<int> & remotePIDs,
+         const Teuchos::RCP<Teuchos::ParameterList>& plist) :
+    debug_(tpetraImportDebugDefault)
   {
-    init (source, target, true, remotePIDs, Teuchos::null);
+    init(source, target, true, remotePIDs, plist);
   }
 
-
   template <class LocalOrdinal, class GlobalOrdinal, class Node>
   Import<LocalOrdinal,GlobalOrdinal,Node>::
-  Import (const Import<LocalOrdinal,GlobalOrdinal,Node>& rhs)
-    : ImportData_ (rhs.ImportData_)
-    , out_ (rhs.out_)
-    , debug_ (rhs.debug_)
+  Import(const Import<LocalOrdinal,GlobalOrdinal,Node>& rhs)
+    : ImportData_(rhs.ImportData_)
+    , out_(rhs.out_)
+    , debug_(rhs.debug_)
   {}
 
   template <class LocalOrdinal, class GlobalOrdinal, class Node>
   Import<LocalOrdinal,GlobalOrdinal,Node>::
-  Import (const Export<LocalOrdinal,GlobalOrdinal,Node>& exporter)
-    : out_ (exporter.out_)
-    , debug_ (exporter.debug_)
+  Import(const Export<LocalOrdinal,GlobalOrdinal,Node>& exporter)
+    : out_(exporter.out_)
+    , debug_(exporter.debug_)
   {
     if (! exporter.ExportData_.is_null ()) {
       ImportData_ = exporter.ExportData_->reverseClone ();
     }
   }
-
-  // This is the "createExpert" version of the constructor
-
+  // cblcbl
+  // This is the "createExpert" version of the constructor to be used with pid/gid pairs obtained from
```

Comment:
This properly goes in the Doxygen documentation, not here.  OK to leave for now, though.